include/lbann/layers/learning/fully_connected.hpp

```diff
@@ -212,7 +212,7 @@ class fully_connected_layer : public learning_layer<TensorDataType> {
 
     // Initialize freeze state
     for (auto&& w : this->get_data_type_weights()) {
-      if (this->m_frozen) {
+      if (this->is_frozen()) {
         w->freeze();
```

Comment:
Note, this file may have nothing to do with this PR.

Reply:
changed back to this->m_frozen, which agrees with the head of develop. I don't know where the change came from -- nothing I did intentionally.